x-pack/platform/plugins/shared/fleet/server/saved_objects/index.ts

```diff
@@ -1167,6 +1168,14 @@ export const getSavedObjectTypes = (
             },
           ],
         },
+        '2': {
+          changes: [
+            {
+              type: 'mappings_addition',
+              addedMappings: {},
```

Comment:
__question:__ what does empty `addedMappings` mean?

Reply:
This "empty migration" is needed as I'm using `dynamic: false` [here](https://github.com/elastic/kibana/pull/208091/files#diff-1fb31be28e4e0aa6f383110895b5b752232fe6d9671a53d9b5ebaf7c239b5cbdR1150). This way new mappings can be added without explicitly declaring them.

Reply:
Ah, I see, thanks for clarifying!